### PR TITLE
Remove all the wrong readme

### PR DESCRIPTION
**Pull request by Hjaltesorgenfrei**

### CODE REVIEW

**Review by Duckapple — APPROVED** [review]
Damn, this is beautiful. Approved!

**Review by jlndk — APPROVED** [review]
Reducing complexity.....

I LOVE IT

File: README.md [hunk 1]
```diff
@@ -5,53 +5,3 @@ A Twitter clone made a long time ago, in a galaxy far, far away. Over the course
 
 **Currently in the middle of a rewrite. This documentation will be rewritten once complete.**
```
Comment (Jakobis) [inline README.md:6]: Arguably not in the middle of a rewrite

**Review by Jakobis — DISMISSED** [review]
Did not r

**Pull request merged**